### DISCUSSION TITLE
Unable to connect to Unifi Protect

### DISCUSSION THREAD

**Discussion by maxnata** [ep1cman/unifi-protect-backup#104] [Q&A]
I have created the new Admin (not "user"?) in Unifi Os with Full Management on Network and Protect.

In the docker installation I have put the username and password and the Protect IP address (it is the default Dream Machine IP).

But when I run the Docker I get the error that Failed to connect to Protect

Any clue? 

Thank you very mjch

**Comment by ep1cman** [comment]
> I have created the new Admin (not "user"?) in Unifi Os with Full Management on Network and Protect.
> 
> 
> 
> In the docker installation I have put the username and password and the Protect IP address (it is the default Dream Machine IP).
> 
> 
> 
> But when I run the Docker I get the error that Failed to connect to Protect
> 
> 
> 
> Any clue? 
> 
> 
> 
> Thank you very mjch

Can you run it with verbosity set to "vv" and post a log?

**Reply by maxnata** [reply]
Worked! A stupid typo... Thanks!
